.github/workflows/ci.yml

```diff
@@ -57,12 +57,39 @@ jobs:
         run: |
           mkdir html
           sphinx-build -b html documentation/library_guide/ html/
+          tar -czvf html.tar.gz html/
       - name: Archive build directory
         uses: actions/upload-artifact@v1
         with:
-            name: onedpl-html-docs-${{ env.GITHUB_SHA_SHORT }}
-            path: html
-
+            name: oneDPL-html-docs-${{ env.GITHUB_SHA_SHORT }}
+            path: html.tar.gz
+  pages:
```

Comment:
Don't forget to add "trigger policy" (we don't need to publish pages on every commit in PRs). 
`github.ref == 'refs/heads/main'`

Reply:
Added